packages/vscode-extension/lib/inspector.js

```diff
@@ -0,0 +1,112 @@
+const { Dimensions, findNodeHandle } = require("react-native");
+const getInspectorDataForViewAtPoint = require("react-native/Libraries/Inspector/getInspectorDataForViewAtPoint");
+
+const getInspectorDataForInstance = (node) => {
```

Comment:
```suggestion
function getInspectorDataForInstance(node) {
```

The same goes for other methods defined here